src/core/cita_bft.rs

```diff
@@ -1989,8 +1989,9 @@ impl Bft {
             self.redo_work();
         }
 
+        let wait_secs = ::std::time::Duration::from_secs(60);
```

Comment:
why 60?

Reply:
A magic number.
Maybe [42] is better!

[42]: https://en.wikipedia.org/wiki/Phrases_from_The_Hitchhiker%27s_Guide_to_the_Galaxy#Answer_to_the_Ultimate_Question_of_Life,_the_Universe,_and_Everything_(42)